### PR TITLE
User module imports

### PR DESCRIPTION
**Pull request by DrChat**

### CODE REVIEW

File: src/xenia/kernel/util/xex2.cc [hunk 1]
```diff
@@ -610,6 +610,7 @@ int xe_xex2_read_image_basic_compressed(const xe_xex2_header_t *header,
     return 1;
   }
   uint8_t *buffer = memory->TranslateVirtual(header->exe_address);
+  std::memset(buffer, 0, total_size); // Quickly zero the contents.
   uint8_t *d = buffer;
   std::memset(buffer, 0, uncompressed_size);
```
Comment (benvanik) [inline src/xenia/kernel/util/xex2.cc]: one of these isn't needed now, right?

Comment (DrChat) [inline src/xenia/kernel/util/xex2.cc]: Portal crashes without this - I'm guessing the BSS section isn't compressed at all.

Comment (benvanik) [inline src/xenia/kernel/util/xex2.cc]: Oh, I mean that there's two memsets of the buffer - only memset(max(total_size, uncompressed_size)) is needed (I don't know which is bigger).

Comment (DrChat) [inline src/xenia/kernel/util/xex2.cc]: Oh wow, you're right - I didn't see that. 2nd one is unnecessary, I'll remove it

**Pull request merged**